WebRoot/templates/calendar/Calendar.template

```diff
@@ -278,6 +278,10 @@
             <td valign=top align=right><$= ZmMsg.sourceLabel $></td>
             <td>
                 <table role="presentation" class="ZRadioButtonTable" width=100%>
+                    <tr>
+                        <td width="5"><input type="radio" name="extCalType" id="${id}_shareRadioGoogle"></td>
+                        <td align=left><label for="${id}_shareRadioGoogle">Google Calendar</label></td>
```

Comment:
Please add this label in translation properties file.

Reply:
Done